doc/man1/openssl-s_client.pod.in

```diff
@@ -656,7 +656,10 @@ For example strings, see L<SSL_CTX_set1_sigalgs(3)>
 =item B<-curves> I<curvelist>
 
 Specifies the list of supported curves to be sent by the client. The curve is
-ultimately selected by the server. For a list of all curves, use:
+ultimately selected by the server.
+
+The list of all supported curves includes named EC parameters as well as X25519
+and X448 or FFDHE groups. For a list of named EC parameters, use:
 
```

Comment:
It's still misleading, sorry. See  SSL_CONF_cmd for more details. FFDHE groups are not curves, but can be provided here.

This parameter corresponds to this IANA registry:
https://www.iana.org/assignments/tls-parameters/tls-parameters.xhtml#tls-parameters-8

Reply:
This is the exact wording proposed by @t8m.  Which wording would you suggest instead?

Reply:
Smth like 
```
The list of all supported groups includes named EC parameters as well as X25519
and X448 or FFDHE groups, and may also include groups implemented in 3rd-party providers.
```
?

Reply:
@t8m, what do you think? Should I change it as suggested above?

Reply:
Yeah, sure